### PR TITLE
Copy the markdownlint summary file out of the container

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

so I can then expose it on the jenkins job.

### CODE REVIEW

**Comment by moxiegirl** [comment]
LGTM

File: jenkins-validate.sh [hunk 1]
```diff
@@ -18,7 +18,9 @@ if git diff --name-only "$PR_BRANCH_BASE..$ghprbActualCommit" | grep "^docs/" ;t
 else 
 	echo "no docs changes"
 	echo '<testsuite tests="1"><testcase classname="markdownlint" name="NoDocChanges"><skipped /></testcase></testsuite>' > junit.xml
-	exit 0
+	if [ -z ${FORCE} ]; then
```
Comment (moxiegirl) [inline jenkins-validate.sh]: What is the -z flag checking for?

Comment (SvenDowideit) [inline jenkins-validate.sh]: http://www.tldp.org/LDP/abs/html/testconstructs.html

is the var set.

**Comment by SvenDowideit** [comment]
oh, how nice. someone deleted Mary's LGTM after i hit merge.

**Pull request merged**